### DISCUSSION TITLE
Add Save Preview

### DISCUSSION THREAD

**Discussion by The-Curious-Geek** [BlafKing/sd-civitai-browser-plus#253] [Ideas]
Can you add a "Save Preview" button that will save a preview image. Or add that functionality to the "Save model info" function. I have a lot of Lora's that I saved before I found your plug and none of them have preview files. When I choose save model info and save images it doesn't create one so the only way I thought was to delete the model and redownload. Looked at the dir as I did one and it just creates a file from (presumably) the first image with name format lorafilename.preview.png (or jpg). I manually changed one of the saved images to this format and it added the preview. Just a thought..

**Comment by yetanotherCoriolis** [comment]
I was looking through the discussions for exactly this!  It would be great if the civitai preview image was saved when "save model info" was pressed.  I currently use this function as a reminder to come back to a model when searching for something, and I don't want to spend gigabytes of download and storage that I may just delete again.  This way all of the previews would be on my machine to easily scan through.

Having the HTML files saved is awesome - kudos! - but they are more detail information than an overview.

**Comment by BlafKing** [comment]
Sry for the late reply!
Kinda overlooked this post, my bad 😅 

Sounds good to me though, I'll probably add a setting which can be toggled to make the `Save model info` button also save a preview

Added to the to-do list! :)

**Comment by latenw** [comment]
Adding another request for this. A simple way to click to update preview image would save a lot of manual work for loads of old LORAs
